screens/Navigation/navHelpers.ts

```diff
@@ -56,6 +56,28 @@ export const getConverseStateFromPath =
     if (pathForState?.startsWith("/")) {
       pathForState = pathForState.slice(1);
     }
+
+    // Handle group invite links
+    if (pathForState?.startsWith("group-invite")) {
+      const url = new URL(`https://${config.websiteDomain}/${pathForState}`);
+      const params = new URLSearchParams(url.search);
+      const inviteId = params.get("inviteId") || url.pathname.split("/")[1];
```

Comment:
Sorry I meant this part, we shouldn't have to change the parameters to make it work here, we should just match what the backend is sending within our linking definition/screen parameters